DEVELOPER.md

```diff
@@ -262,7 +262,7 @@ Here are the steps on how to override them:
 
       1. AWS S3 BUCKET
 
-         If you have an MicroFocus account for AWS, you can configure your environment so that it uses AWS instead of minio.  There are pre-existing buckets setup for developers to use.  Here are the steps to set that up:
+         If you have an AWS account, you can configure your environment so that it uses AWS instead of minio.  There are pre-existing buckets setup for developers to use.  Here are the steps to set that up:
```

Comment:
> There are pre-existing buckets setup for developers to use.

Maybe it should say, "The s3 bucket must already be created prior to running the tests".

Reply:
Ah this for any developer so it has to be the most general possible. right?

Reply:
So "pre-existing buckets setup for developers" are only for internal use?

Reply:
Right.  That sentence relates to the internal buckets Vertica uses for testing.  Best to have it more general.